apps/i18n/common/en_us.json

```diff
@@ -640,6 +640,7 @@
   "dropletBlock_subtractOperator_signatureOverride": "Subtract operator",
   "dropletBlock_whileBlock_description": "Creates a loop consisting of a conditional expression and a block of statements executed for each iteration of the loop. The loop continues to execute as long as the condition evaluates to true",
   "dropletBlock_whileBlock_signatureOverride": "while loop",
+  "droplet_parsing_error": "Error parsing your code. Check for a curly bracket that isn't attached to a function, loop, or conditional.",
```

Comment:
are there any other parsing errors that would lead to this message? Is this message based on a guess of what the common issues are? I think it would be really confusing to get this error message if the issue turns out not to be a block statement.
Also nit: I think "curly brace" is more common than "curly bracket"?
another nit: the word "attached" strikes me as a little odd here.

Reply:
> are there any other parsing errors that would lead to this message?

It's possible.  The exact area that throws this error is here:
https://github.com/droplet-editor/droplet/blob/ee759e832a06a9989fa59de5ff14d760838821d2/src/parser.coffee#L435-L444
^ The comments and code here seem to be regarding indents, but the errors we see don't have anything to do with indents. Would you recommend just passing the error directly to the user?

> Also nit: I think "curly brace" is more common than "curly bracket"?

I think it depends on what field you are in. I chose "curly bracket" because it has a wikipedia page.

> another nit: the word "attached" strikes me as a little odd here.

Any recommendation for a replacement?

Reply:
Yes, I think passing a more general but definitely-correct error message is better than giving potentially-incorrect information.

Can't think of anything better, so attached is probably fine (if you end up keeping an error message related to curly braces at all)